src/main/java/software/amazon/awssdk/crt/http/HttpClientConnectionManagerOptions.java

```diff
@@ -15,14 +15,14 @@
  */
 public class HttpClientConnectionManagerOptions {
     public static final int DEFAULT_MAX_BUFFER_SIZE = 16 * 1024;
-    public static final int DEFAULT_MAX_WINDOW_SIZE = Integer.MAX_VALUE;
+    public static final long DEFAULT_MAX_WINDOW_SIZE = Integer.MAX_VALUE;
```

Comment:
trivial: should this be changed to long max?

Reply:
I am not sure if that will work on 32-bit systems where the maximum value of size_t is 32 bits.